README.md

```diff
@@ -67,28 +67,69 @@ npm install cancel-stuck-transactions
 
 ## API
 
-### `CancelStuckTransactions({ store, list, resolve, log, sendTransaction })`
+### `StuckTransactionsCanceller({ store, log, sendTransaction })`
+
+```js
+import { StuckTransactionsCanceller } from 'cancel-stuck-transactions'
+```
 
 Options:
 
-- `store`: `({ hash: string, timestamp: string, from: string, maxPriorityFeePerGas: bigint, nonce: number }) -> Promise`
-- `list`: `() -> Promise<{ hash, timestamp, from, gasPremium, nonce }[]>`
-- `resolve`: `(hash) -> Promise`
-- `log`: `str -> null`
-- `sendTransactions`: `(tx) -> Promise<tx>`
+- `store`:
+  - `store.add`: `({ hash: string, timestamp: string, from: string, maxPriorityFeePerGas: bigint, nonce: number }) -> Promise`
+  - `store.list`: `() -> Promise<{ hash: string, timestamp: string, from: string, maxPriorityFeePerGas: bigint, nonce: number }[]>`
+  - `store.remove`: `(hash: string) -> Promise`
+- `log`: `(string) -> null`
+- `sendTransactions`: `(Transaction) -> Promise<Transaction>`
```

Comment:
Since the function works with a single transaction, I think the name should also use a singular form.

```suggestion
- `sendTransaction`: `(Transaction) -> Promise<Transaction>`
```

Reply:
This was only a docs typo :)